::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
: Copyright (c) 2017 Mirantis Inc., Enea AB and others.
:
: All rights reserved. This program and the accompanying materials
: are made available under the terms of the Apache License, Version 2.0
: which accompanies this distribution, and is available at
: http://www.apache.org/licenses/LICENSE-2.0
::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::::
From: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
Date: Sat, 16 Dec 2017 21:14:35 +0100
Subject: [PATCH] Update MaaS PXE config

- fix DHCP range which is opt-out, not opt-in as previously assumed;
- define PXE/admin address for Salt Master node;
- drop obsolete opnfv_infra_maas_pxe_address;

JIRA: FUEL-316

Signed-off-by: Alexandru Avadanii <Alexandru.Avadanii@enea.com>
---
 config/installers/fuel/pod_config.yml.j2 | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/config/installers/fuel/pod_config.yml.j2 b/config/installers/fuel/pod_config.yml.j2
index 28c6188..2e9dbd3 100644
--- a/config/installers/fuel/pod_config.yml.j2
+++ b/config/installers/fuel/pod_config.yml.j2
@@ -40,6 +40,7 @@ parameters:
 {%- endif %}

     opnfv_infra_config_address: {{ net_mgmt | ipaddr_index('100') }}
+    opnfv_infra_config_pxe_address: {{ net_admin | ipaddr_index('2') }}
     opnfv_infra_maas_node01_address: {{ net_mgmt | ipaddr_index('3') }}
     opnfv_infra_maas_node01_deploy_address: {{ net_admin | ipaddr_index('3') }}
     opnfv_infra_kvm_address: {{ net_mgmt | ipaddr_index('140') }}
@@ -48,9 +49,8 @@ parameters:
     opnfv_infra_kvm_node03_address: {{ net_mgmt | ipaddr_index('143') }}

     opnfv_infra_maas_pxe_network_address: {{ net_admin }}
-    opnfv_infra_maas_pxe_address: {{ net_admin | ipaddr_index('3') }}
-    opnfv_infra_maas_pxe_start_address: {{ net_admin | ipaddr_index('5') }}
-    opnfv_infra_maas_pxe_end_address: {{ net_admin | ipaddr_index('250') }}
+    opnfv_infra_maas_pxe_start_address: {{ net_admin | ipaddr_index('1') }}
+    opnfv_infra_maas_pxe_end_address: {{ net_admin | ipaddr_index('3') }}

     opnfv_openstack_gateway_node01_address: {{ net_mgmt | ipaddr_index('124') }}
     opnfv_openstack_gateway_node02_address: {{ net_mgmt | ipaddr_index('125') }}
